doc/examples/edges/plot_edge_filter.py

```diff
@@ -88,3 +88,73 @@
 
 plt.tight_layout()
 plt.show()
+
+######################################################################
+# As in the example above, we illustrate the rotational invariance
+# of the filters in this example. We see an rotationally invariant image
+# along with the angle of its analytical gradient in the top row.
+# The other two rows contain the difference between the different gradient
+# approximations (Sobel, Prewitt, Scharr & Farid) and the analytical gradient
+# Farid & Simoncelli derivative filters are most rotationally invariant, but
+# require a 5x5 kernel for this advantage to unfold.
+#
+# .. [1] https://en.wikipedia.org/wiki/Sobel_operator#Alternative_operators
+#
+# .. [2] Farid, Hany, and Eero P. Simoncelli. "Differentiation of discrete
+#     multidimensional signals."
+#     IEEE Transactions on image processing 13.4 (2004): 496-508.
+#
+# .. [3] https://en.wikipedia.org/wiki/Image_derivatives
+# #Farid_and_Simoncelli_Derivatives
+
+
+x, y = np.mgrid[-10:10:255j, -10:10:255j]
+img = np.sin(x ** 2 + y ** 2)
+
+imgx = 2 * x * np.cos(x ** 2 + y ** 2)
+imgy = 2 * y * np.cos(x ** 2 + y ** 2)
+
+angle = lambda dx, dy: np.mod(np.arctan2(dy, dx), np.pi)
+true_angle = angle(imgx, imgy)
+
+angle_farid = angle(farid_h(img), farid_v(img))
+angle_sobel = angle(sobel_h(img), sobel_v(img))
+angle_scharr = angle(scharr_h(img), scharr_v(img))
+angle_prewitt = angle(prewitt_h(img), prewitt_v(img))
+
+diff_farid = np.abs(true_angle - angle_farid)
+diff_sobel = np.abs(true_angle - angle_sobel)
+diff_scharr = np.abs(true_angle - angle_scharr)
+diff_prewitt = np.abs(true_angle - angle_prewitt)
+
+fig, axes = plt.subplots(nrows=3, ncols=2, sharex=True, sharey=True,
+                         figsize=(8, 8))
+ax = axes.ravel()
+
+ax[0].imshow(img, cmap=plt.cm.gray)
+ax[0].set_title('Original image')
+
+ax[1].imshow(true_angle, cmap=plt.cm.hsv)
+ax[1].set_title('Analytical gradient angle')
+
+ax[2].imshow(diff_sobel, cmap=plt.cm.jet, vmin=0, vmax=0.02)
+ax[2].set_title('Sobel')
+
+ax[3].imshow(diff_prewitt, cmap=plt.cm.jet, vmin=0, vmax=0.02)
+ax[3].set_title('Prewitt')
+
+ax[4].imshow(diff_scharr, cmap=plt.cm.jet, vmin=0, vmax=0.02)
+ax[4].set_title('Scharr')
+
+cax = ax[5].imshow(diff_farid, cmap=plt.cm.jet, vmin=0, vmax=0.02)
+ax[5].set_title('Farid')
+
+fig.subplots_adjust(right=0.8)
+cbar_ax = fig.add_axes([0.85, 0.10, 0.05, 0.50])
+fig.colorbar(cax, cax=cbar_ax)
```

Comment:
fig.colorbar(cax, cax=cbar_ax, ticks=[0, 0.01, 0.02])

Reply:
In the rendered html page, the colorbar overlaps with the plots, so it should be made as thin as possible